project/target/portfolio-1/categories.html

```diff
@@ -19,13 +25,20 @@ <h1 id = "page-title">
         <table id = "categories-table" class = 'list-table'>
             <!-- TODO: Create links for the different names -->
             <!-- TODO: Customize CSS -->
-            <tr><td onclick="window.location.href = 'video-games-profiles.html';">Video games</td></tr>
+            <tr><td onclick="relocateUser('video-games-profiles.html');">Video games</td></tr>
             <tr><td onclick="window.location.href = 'index.html';">Sports</td></tr>
             <tr><td onclick="window.location.href = 'index.html';">Fashion</td></tr>
             <tr><td onclick="window.location.href = 'index.html';">Food</td></tr>
             <tr><td onclick="window.location.href = 'index.html';">Music</td></tr>
             <tr><td onclick="window.location.href = 'index.html';">Travel</td></tr>
         </table>
     </div>
+    <script>
+        function assignUsernameValue(){
```

Comment:
Since this is used a bunch of times, it might be good to pull it out into its own script instead and make it generic. Let me know if you have any questions on doing that.